### PR TITLE
Fix routes

### PR DESCRIPTION
**Pull request by majkus522**
Fix access to deeper routes, e.g. weekly-boss

### CODE REVIEW

**Review by SteeledSlagle13 — APPROVED** [review]
nice

**Review by man90es — APPROVED** [review]

File: src/routes/index.ts [hunk 1]
```diff
@@ -77,43 +63,62 @@ router.get('/:type/all', async (ctx) => {
...
-router.get('/:type/:id/list', async (ctx) => {
-  const { type, id } = ctx.params;
-
+router.get('/:type*/list', async (ctx) => {
+  let { type } = ctx.params;
+  const id = type.substring(type.lastIndexOf("/") + 1);
+  type = type.substring(0, type.lastIndexOf("/"));
   try {
+    if(await containsFolders(ctx.params.type))
+      throw new Error(`No images for ${ctx.params.type}/lista exist`);
```
Comment (M4GICB) [inline src/routes/index.ts]: what does `lista` mean here? should that be there?

Comment (majkus522) [reply src/routes/index.ts:72]: Some leftover from debugging.
Already fixed.

Comment (majkus522) [reply src/routes/index.ts]: Just a typo

**Review by M4GICB — APPROVED** [review]

**Pull request merged**